pytorch_lightning/callbacks/quantization.py

```diff
@@ -91,45 +92,58 @@ class QuantizationAwareTraining(Callback):
 
     .. warning:: ``QuantizationAwareTraining`` is in beta and subject to change.
 
+    The model set for quantization can appear in one of these stages::
 
-    Args:
+        TRAINER TRANSITIONS
+                          ( on_fit_start )               ( on_fit_end )
+        MODEL STATES
+            vanilla model      --->       QuantAwareTrain     --->     quantized model
+                               /                |
+                    Trainer --/                 |
+            ( resume_from_checkpoint )          v
+                       ^                  QAT checkpoints
+                       |----------------------/
```

Comment:
This doesn't look very easy to read at all...

Reply:
any other suggestions?

Reply:
My suggestion is don't try to draw a graph in plaintext 😂 

Rewrite it in pseudocode

Reply:
I would rather have some chart the pseudocode...
@t-vi @aribornstein what do you think?

Reply:
Yes, this is un-clear with me either. Missing what are the on_fit_start, on_fit_end consequences on the model.

Reply:
on_fit_start, on_fit_end are Trainer hooks so what else do you suggest, otherwise I ll just delete it :rabbit: